apps/wfo-ui/pages/forms.tsx

```diff
@@ -0,0 +1,25 @@
+import { EuiHorizontalRule, EuiPageHeader, EuiSpacer } from '@elastic/eui';
+import React from 'react';
+import CreateForm from '@orchestrator-ui/orchestrator-ui-components/src/components/WFOForms/CreateForm';
+
+export function FormsPage() {
```

Comment:
Just to check: we do want this one for now in the MVP branch?

Reply:
AFAIK. It's fine, it shows an example implementation